pkgs/top-level/all-packages.nix

```diff
@@ -16580,6 +16580,8 @@ with pkgs;
 
   koka = haskell.lib.compose.justStaticExecutables (haskellPackages.callPackage ../development/compilers/koka { });
 
+  kokkos = callPackage ../development/libraries/kokkos { };
```

Comment:
Please move `pkgs/development/libraries/kokkos/default.nix` to  `pkgs/by-name/ko/kokkos/package.nix`.